apps/user_ldap/lib/LDAP.php

```diff
@@ -320,7 +320,15 @@ private function preFunctionCall(string $functionName, array $args): void {
 		$this->curArgs = $args;
 
 		if ($this->dataCollector !== null) {
-			$args = array_map(fn ($item) => (!$this->isResource($item) ? $item : '(resource)'), $this->curArgs);
+			$args = array_map(function ($item) {
+				if ($this->isResource($item)) {
+					return '(resource)';
+				}
+				if (isset($item[0]['value']['cookie'])) {
+					$item[0]['value']['cookie'] = "";
```

Comment:
```suggestion
					$item[0]['value']['cookie'] = "*opaque cookie*";
```
As an empty cookie has a special meaning in some cases?

Reply:
Well you also need to change the if to only replace non-empty strings if you go for this.

Reply:
sounds like a good idea